backend/.github.backup/workflows/test.yml

```
@@ -0,0 +1,269 @@
+        
+    - name: 📦 Install dependencies
+      run: npm ci
+      
+    - name: 🏗️ Generate Prisma Client
+      run: npx prisma generate
+      
+    - name: 🗄️ Run Database Migrations
+      run: npx prisma migrate deploy
+      env:
+        DATABASE_URL: postgresql://testuser:testpass@localhost:5433/test_taskmanagement?schema=public
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)